fossevents/templates/email/event_moderators_email.html

```diff
@@ -0,0 +1,8 @@
+{% load staticfiles i18n %}
+
+Hi,
+
+An event has been created. Please visit this <a href="#">url</a> for reviewing.
+
+Sincerely,
+The {{ site_name }} Team
```

Comment:
Let's keep the footer consistent and can be placed in the base.html for emails. All the emails should extend base.html. All *.html must contain html markups.